src/main/java/io/quarkus/gizmo2/impl/Util.java

```diff
@@ -243,36 +237,14 @@ protected MethodDesc computeValue(final Class<?> type) {
                 }
             }
             if (sam == null) {
-                throw new IllegalArgumentException("No abstract method found on " + type);
+                throw new IllegalArgumentException("No SAM found on " + type);
```

Comment:
BTW, I originally changed the wording from "SAM" to "abstract method" because:

1. "SAM" is a Java/JVM engineer colloquialism that doesn't necessarily have to be well known to ordinary population.
2. There are 2 reasons for why you cannot find a SAM: there's no abstract method, or there's more than one abstract method. Since we throw different error messages in these situations, I find it appropriate to mention "abstract method" in both of them (or none, but why just in one and not the other?).